tests/unit/tests_standard_lib/test_addon_parser/test_tags_parser.py

```diff
@@ -1,31 +1,34 @@
+import os
+from unittest.mock import patch, mock_open
+
 import pytest
-from unittest.mock import patch, PropertyMock
+
 from pytest_splunk_addon.standard_lib.addon_parser.tags_parser import (
     TagsParser,
 )
 
+TEST_TAGS_WITHOUT_STANZA_NAME = """tags_positive_event = enabled
+tags_disabled_event = disabled
+"""
 
-output_to_build = {
-    "eventtype=fiction_for_tags_positive": {
-        "tags_positive_event": "enabled",
-        "tags_disabled_event": "disabled",
-    },
-    "source=%2Fopt%2Fsplunk%2Fvar%2Flog%2Fsplunk%2Fsplunkd.log": {
-        "tags_positive_event": "enabled",
-        "tags_disabled_event": "disabled",
-    },
-}
+
+@pytest.fixture()
+def default_tags_parser():
```

Comment:
all tests we do here are executing conf-parser module (no mocking), but since conf-parser is quite small it's good enough for me